### ISSUE TITLE
Crash : create a task with an empty name

### ISSUE THREAD

**Issue by jdorel** [kdheepak/taskwarrior-tui#185]
When trying to add a task with a name, the application crash :
```
Backtrace (most recent call first):
  File "<unknown>", line 0, in __libc_start_main

The application panicked (crashed).
  called `Option::unwrap()` on a `None` value
in src/app.rs, line 1629
thread: main
```

**To Reproduce**

Press `a` to add a task, then press `Enter`. You can also add a tag or a special field (like due:today), the crash will also happen.

**Environment (please complete the following information):**

- Operating System: Linux
- Installation: arch
- taskwarrior-tui version: 0.13.11
- taskwarrior version: 2.6.0

**Comment by sergi-ko** [comment]
@jdorel this is related to https://github.com/kdheepak/taskwarrior-tui/issues/183 and has been fixed in 0.13.12

**Comment by kdheepak** [comment]
Thanks @sergi-ko for responding so quickly about this! Yes @jdorel, this is resolved in the latest versions. Sorry about the inconvenience!
